src/routes/recipe.ts

```diff
@@ -38,4 +40,21 @@ router.get(
   validate,
   getRecipesLikeByUserId
 );
+router.delete(
+  "/:id",
+  protectedRoute,
+  getParamIdValidator(),
+  validate,
+  (req, res) => {
+    RecipeModel.deleteRecipeById({
+      recipeId: req.recipe.id,
+      userId: req.user.id,
+    })
+      .then((recipeData) => {
+        res.status(200);
+        return recipeData;
```

Comment:
Here you're sending a status 200 repsonse, but you're not sending the updated recipeData